SWIG/ratehelpers.i

```diff
@@ -330,7 +330,9 @@ class OISRateHelper : public RateHelper {
             Pillar::Choice pillar = Pillar::LastRelevantDate,
             Date customPillarDate = Date(), 
             RateAveraging::Type averagingMethod = RateAveraging::Compound,
-            ext::optional<bool> endOfMonth = ext::nullopt);
+            ext::optional<bool> endOfMonth = ext::nullopt,
+            ext::optional<Frequency> fixedPaymentFrequency = ext::nullopt,
```

Comment:
`optional<Frequency>` needs a couple of typemaps to work that so far are not there, see other `optional` parameters in the SWIG files.